zul/src/archive/web/js/zul/tab/Tabpanels.js

```diff
@@ -107,14 +107,15 @@ zul.tab.Tabpanels = zk.$extends(zul.Widget, {
 		this._fixWidth();
 	},
 	_fixWidth: function () {
-		var parent = this.parent.$n();
+		var parent = this.parent.$n(),
+			$parent = jq(parent);
 		if (this.__width || !zk(parent).isRealVisible())
 			return;
 
-		var width = parent.offsetWidth,
+		var width = $parent.width(),
```

Comment:
Use content width of parent instead of offsetWidth (content + border + padding)
The border and padding shouldn't be computed.